contents/docs/privacy/hipaa-compliance.md

```diff
@@ -37,31 +37,28 @@ There are downsides to these two solutions:
 
 PostHog offers a third approach without either of these downsides: hosting the product analytics systems yourself.
 
-## How to set PostHog up for HIPAA compliant analytics
-
-PostHog enables you to self-host on your own infrastructure and maintain full control of the data. This means you don't need to anonymize the data, nor do you need to set up a Business Associate Agreement with PostHog because you never need to send any Protected Health Information (PHI) to us in the first place. The data stays on your systems, in its original form. 
-
-You may need to sign a BAA with your hosting provider, but major providers such as Google and AWS offer these for free.
-
-### Step 1: Choose a hosting provider
+## Does PostHog offer a BAA for PostHog Cloud?
 
-We recommend hosting PostHog on your own infrastructure. If you’re leveraging a private cloud you will need a Business Associate Agreement with your provider first. These are commonly and easily available with services such as [Amazon Web Services](https://aws.amazon.com/compliance/hipaa-compliance/), [Google Cloud Platform](https://cloud.google.com/security/compliance/hipaa), [Microsoft Azure](https://docs.microsoft.com/en-us/azure/compliance/offerings/offering-hipaa-us) and many more, often for free.
+Yes. If you're interested in a BAA for HIPAA compliance, please contact [sales@posthog.com](mailto:sales@posthog.com) with information about your requirements.  
 
-### Step 2: Deploy PostHog
+## How to set PostHog up for HIPAA compliance
 
-Deploying PostHog onto your own infrastructure is straightforward and we provide support to solve any issues you encounter. You can follow our [standard deployment guides](https://posthog.com/docs/self-host) to get started, or [arrange a demo](https://posthog.com/book-a-demo) to see it in action first.
+> We strongly recommend teams which need HIPAA compliance use PostHog Cloud under a BAA, rather than the legacy self-hosted product or Hobby versions. 
 
-### Step 3: Security configuration
+The best way to use PostHog in a way which is HIPAA compliant is with a BAA covering a PostHog Cloud instance. This ensures that PostHog remains scalable and that you have access to all premium features and security enhancements. 
 
-When setting up a PostHog instance **we strongly recommend that you use HTTPS** to secure data in transmission, whether or not your instance has access to the wider internet. We also have a [guide for securing PostHog](https://posthog.com/docs/self-host/configure/securing-posthog) which you should follow to further protect your instance.
+However, it is possible to host PostHog yourself using either our legacy self-hosted product, or our open-source hobby deployment. It is important to note that the legacy product is no longer supported, and that the hobby deployment is only suitable for smaller event volumes and provided without guarantee. Both versions lack many of the advanced features available under PostHog Cloud. 
```

Comment:
I don't think we should mention the "legacy" product here. Afaik, you can't get a license anyway so is it event possible?

Reply:
```suggestion
However, it is possible to host PostHog yourself using either our open-source hobby deployment. It is important to note that the hobby deployment is only suitable for smaller event volumes and provided without guarantee. It also lacks many of the advanced features available under PostHog Cloud. 
```